.dockerignore

```diff
@@ -1,5 +1,10 @@
+# DO NOT ADD ANY CHECKED-IN FILES TO THIS LIST!
+# During the build, we look for tracked files with changes and if we find any,
+# we add a "dirty" tag to the version string. Therefore if any checked-in file
```

Comment:
Just out of curiosity, which build or step will cause this kind of failure?

Reply:
just running `make` on the main branch